data/json/items/tool/workshop.json

```diff
@@ -1078,5 +1078,19 @@
     "color": "light_gray",
     "qualities": [ [ "WRENCH", 2 ], [ "WRENCH_FINE", 1 ] ],
     "flags": [ "BELT_CLIP" ]
+  },
+  {
+    "id": "makeshift_hose",
+    "type": "TOOL",
+    "name": "leather hose",
+    "description": "This is a hose made out of leather. Watertight and flexiable, it can be used to siphon fuel from a vehicle tank. However, it's not as elastic a rubber hose.",
```

Comment:
```suggestion
    "description": "This is a hose made out of leather.  Watertight and flexiable, it can be used to siphon fuel from a vehicle tank.  However, it's not as elastic a rubber hose.",
```